### ISSUE TITLE
Be more careful with replicating a foaf if some of our friends are blocking them

### ISSUE THREAD

**Issue by Powersource** [ssbc/ssb-friends#57]
See e.g. https://github.com/ssbc/ssb-friends/issues/54 for an example of what this would help with (but also a bunch more)

**Comment by clehner** [comment]
Also discussed at [`%ugLvg+0JSUlRwxENLN+kwfW2IG1BbUGPIrgb7mUnx7o=.sha256`](https://viewer.scuttlebot.io/%25ugLvg%2B0JSUlRwxENLN%2BkwfW2IG1BbUGPIrgb7mUnx7o%3D.sha256), proposing a configurable threshold at which our follows blocking a feed overrides our follows following that feed.

**Comment by Powersource** [comment]
Yeah exactly that. Cel would you be up for some pair programming with me to take a stab at it?

**Comment by clehner** [comment]
Sure, @Powersource

**Comment by staltz** [comment]
Is this planned to happen? In Manyverse we're planning to implement trustnet for SSB soon, and it would have the same goal as this issue.

**Comment by Powersource** [comment]
@staltz i think we had a call but i don't think any code came out of it. trustnet is probably a better solution

**Comment by staltz** [comment]
Ok, I would close this issue then since it's not so actionable. But I'll leave that decision to you.

**Comment by Powersource** [comment]
i think it's actionable, it would get closed by implementing trustnet :)
